src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -547,10 +549,11 @@ internal override ValueTask ConnectAsync(CancellationToken cancellationToken = d
                 throw new Exception($"{nameof(ConnectAsync)} must not be called on a connection obtained from a listener.");
             }
 
-            (string address, int port) = _remoteEndPoint switch
+            string? targetHost = _targetHost;
```

Comment:
```suggestion
            string targetHost;
```
combined with suggestion on line 583

Reply:
updated. seems more compact but functionally same.